node-api/handlers/beacon/routes.go

```diff
@@ -184,25 +193,49 @@ func (h *Handler) RegisterRoutes(
 			Handler: h.NotImplemented,
 		},
 		{
+			// Deprecated endpoint.
 			Method:  http.MethodGet,
 			Path:    "/eth/v1/beacon/pool/attestations",
 			Handler: h.NotImplemented,
 		},
 		{
+			// Deprecated endpoint.
 			Method:  http.MethodPost,
 			Path:    "/eth/v1/beacon/pool/attestations",
 			Handler: h.NotImplemented,
 		},
 		{
+			Method:  http.MethodGet,
```

Comment:
@nidhi-singh02 looks like some endpoints are missing like `/eth/v1/beacon/states/{state_id}/validator_identities`.
What is the principle around including some and not others?

Reply:
Also some are not relevant anymore like `blinded_blocks` one?

Reply:
Would either add all from spec and then mark as deprecated OR only include ones relevant for us. okay either way

Reply:
I saw block rewards missing in route so added it. Then noticed few endpoints missing. 
Fair point, will go through the whole beacon list and add all the missing endpoints. I would prefer adding a comment as deprecated.